frontend/src/scenes/insights/views/LineGraph/LineGraph.tsx

```diff
@@ -649,12 +651,13 @@ export function LineGraph_({
                     ticks: {
                         display: !hideYAxis,
                         ...tickOptions,
-                        precision,
+                        ...(yAxisScaleType !== 'log10' && { precision }), // Precision is not supported for the log scale
                         callback: (value) => {
                             return formatPercentStackAxisValue(trendsFilter, value, isPercentStackView)
                         },
                     },
                     grid: gridOptions,
+                    type: yAxisScaleType === 'log10' ? 'logarithmic' : 'linear',
```

Comment:
Log scale graphs don't show zero values, see the [screenshot](https://github.com/PostHog/posthog/assets/6707908/2b1f937e-3bd4-4c09-a5f0-95df6ec388a5). This can be fixed (as per [this thread](https://github.com/chartjs/Chart.js/issues/9629)), I wonder if it's needed?

Reply:
Hmm, I think complete omission – as in the screenshot – is pretty confusing. But Grafana seems to do this pretty well! They seem to treat 0 values as essentially negative infinite on the log scale:

<img width="779" alt="Screenshot 2024-06-19 at 15 00 30" src="https://github.com/PostHog/posthog/assets/4550621/9b05b2d2-a616-40c8-9498-518857717b96">

Can we pull this off?

Reply:
The only workaround I found that might work is
1. Replace zeroes with 1e-12
2. Set minimum of the graph to the minimum value among the trend values (so that the graph doesn't show 1e-12, like [here](https://github.com/chartjs/Chart.js/issues/9629#issuecomment-915083652))
3. Figure out how to show "0" instead of "1e-12" when hovering, [this person](https://github.com/chartjs/Chart.js/issues/9629#issuecomment-933141998) couldn't do it.

Do you think it's worth trying? Because that sounds like a lot and the result is not guaranteed.

Reply:
Played with this a bit and got Grafana-like results (pushed a commit):
### Before
<img width="1263" alt="Screenshot 2024-06-20 at 14 03 24" src="https://github.com/PostHog/posthog/assets/4550621/f8a99564-b482-4120-9d1c-e8100241cf83">

### After
<img width="1264" alt="Screenshot 2024-06-20 at 14 04 34" src="https://github.com/PostHog/posthog/assets/4550621/f5921ff9-fa8a-421f-8f69-df8834efcdb1">

Reply:
Damn, thanks!